Microsoft.Toolkit.Win32/Microsoft.Toolkit.Sample.Wpf.App/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Windows;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Microsoft.Toolkit.Sample.Wpf.App")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Microsoft.Toolkit.Sample")]
+[assembly: AssemblyCopyright("Copyright ©  2018")]
```

Comment:
Need to be filled out properly

Reply:
Looking thru this, I'm not sure any of the projects in the Win32 solution have this info filled out properly.  I'll try to find a good example and propagate the pattern.